configure.ac

```diff
@@ -682,6 +682,7 @@ AC_LANG_POP([C++])
 AC_CONFIG_MACRO_DIRS([m4])
 
 AC_CONFIG_FILES([Makefile				\
+		 opencryptoki.pc			\
```

Comment:
Doesn't work with `make prefix=/foo/bar` which is allowed by autotools.

Reply:
So what would be the solution (if any)?

Reply:
Either document that this is not supported or generate the .pc-file during make.  Not sure if anybody uses this feature.  Never found a use case myself and always wondered why I can configure a prefix directory and the decide during make to use a different one.  But yes, all variables (`prefix`, `libdir`, ...) can be overwritten when invoking `make`.  But to make things even worse, I think you can even overwrite them when doing a `make install`...

Summing up, I think documenting that his variable overwriting during make is not supported might be the way to go.

Reply:
Created issue https://github.com/opencryptoki/opencryptoki/issues/577 so that we don't forget about this.